org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -65,6 +180,46 @@
             contentTypeId="org.eclipse.wildwebdeveloper.json"
             scopeName="source.json">
       </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.bowerrc"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.bower"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.jsconfig"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.tsconfig"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.package"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.omnisharp"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.typings"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.lerna"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.tslint"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
+      <scopeNameContentTypeBinding
+            contentTypeId="org.eclipse.wildwebdeveloper.json.eslintrc"
+            scopeName="source.json">
+      </scopeNameContentTypeBinding>
```

Comment:
Same as above, association from the base type org.eclipse.wildwebdeveloper.json should be enough and not require extra definition here.